File: app/src/Models/CourseClass.ts
class CourseClass extends BaseModel {
    public component: string; // LEC
    public section: string; // B1
    public location: string;
    public day: string; // MWF, TR
    public startTime: string;
    public endTime: string;
    public class: number; // 87582
    public courseID: number; // 001344
    public term: number; // 16660
    public capacity: number; // 255
    public classStatus: string; // A
    public enrollStatus: string; // O
    public startDate: string; // YYYY-MM-DD
    public endDate: Date;
    public examDate: Date;
    public examStartTime: Date;
    public examEndTime: Date;
    public examLocation: string;
    public examStatus: string;
    public campus: string;
    public classNotes: string; // Restricted to Engineering students. Closed to web
    public classType: string; // E
    public consent: string;
    public gradingBasis: string;
    public instructionMode: string; // In Person
    public instructorID: string; // wjaeger
    public session: string; // Regular academic session
    public units: number; // 3.0

    public toString() {
        return "";
    }
}

File: Models/CourseClass.cs
using System;


namespace BetterBearTracks.Models
{
	public class CourseClass {
		public string component { get; set; }
		public string section { get; set; }
		public string location { get; set; }
		public string day { get; set; }
		public DateTime startTime { get; set; }
		public DateTime endTime { get; set; }
		public int classID { get; set; }
		public int courseID { get; set; }
		public int term { get; set; }
		public int capacity { get; set; }
		public char classStatus { get; set; }
		public char enrollStatus { get; set; } 
		public DateTime startDate { get; set; }
		public DateTime endDate { get; set; }
		public DateTime examDate { get; set; }
		public DateTime examTime { get; set; }
		public string examLocation { get; set; }
		public string examStatus { get; set; }
		public string campus { get; set; }
		public string classNotes { get; set; }
		public string classType { get; set; }
		public string consent { get; set; }
		public string gradingBasis { get; set; }
		public string instructionMode { get; set; }
		public string instructorID { get; set; }
		public string session { get; set; }
		public float units { get; set; }
	}
}
